### PR TITLE
WIP: Improve look of documentation

### PR DESCRIPTION
**Pull request by TheZoq2**
The documentation felt a bit too cluttered for me, so I changed it a bit. This is what it looks like rendered:

image

I'm not a huge fan of the colors on the doc headers either, but that might be more of a personal preference. In any case, i'll try to come up with an alternataive

### CODE REVIEW

**Comment by TheZoq2** [comment]
image
A proposal for reducing the amount of color. To me, this draws the eyes towards the actual text rather than the header bars.

**Comment by TheZoq2** [comment]
I should probably focus more on the docs content, but this is a lot more fun :)

- Limited maximum width of content to avoid super long lines
- TOC to a floating sidebar if there is enough space
- Re-ordered documentation to put "What is RIBS" on top

Peek 2020-09-20 12-00

**Comment by FredTheDino** [comment]
Nice! Looks a lot easier to parse now! Nice PR.

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ pip install -r requirements.txt
 
 ## Documentation
 
-The documentation for RIBS can be found at [tbd]
+The documentation for RIBS can be found at https://lithekod.github.io/snake-ribs/
```
Comment (FredTheDino) [inline README.md:26]: Shouldn't this be a link?

**Comment by TheZoq2** [comment]
One final thing, removed the argument list from the TOC to make that easier to parse :)

**Comment by TheZoq2** [comment]
Hang on, something broke when I did something and now text is super small, going to look into it

**Review by FredTheDino — APPROVED** [review]
Great PR!

Would be nice with:
  - Some padding on the bottom of the ToC
  - A slightly larger monospace font.
  - Some way to differentiate code blocks from the other code, a different background or a border or something.

File: README.md (hunk 1 above)
Comment (TheZoq2) [reply README.md:26]: Yes, I assumed links were detected automagically when I wrote that, i'll take care of it

**Comment by TheZoq2** [comment]
Looks like html got confused by the `<code>` tags being present in the ids and names of elements. Eventually it had enough and just filled the whole document with them :) Should be fixed now

**Comment by TheZoq2** [comment]
Never mind, still not fixed :scream:

File: README.md (hunk 1 above)
Comment (TheZoq2) [reply README.md:26]: Oh wait, this is markdown so it should be automatically made a link I think

**Comment by TheZoq2** [comment]
Ok, fixed now (I think)

Making my own todo list:

- [x] Some padding on the bottom of the ToC
- [x] A slightly larger monospace font.
- [x] Some way to differentiate code blocks from the other code, a different background or a border or something.
- [x] Nicer looking links in the TOC
- [ ] Maybe highlight default arguments and keyword arguments in the titles
- [x] Darker code block background
- [ ] Fix ugly spacing after description blocks

**Comment by TheZoq2** [comment]
image

Moved the gray color to the code examples only. I think it's an improvement but I'm not sure

**Comment by FredTheDino** [comment]
I like it, makes them easier to parse! :+1:

**Comment by sornas** [comment]
I agree with having grey only for code but I think it can be a bit darker.

**Comment by TheZoq2** [comment]
image

Updated table of contents to get rid of ugly visited link color, improve font size and add a hover effect.

I also added a bit more spacing between headers and content. Next i'll look into fixing the ugly spacing after some descriptions

image

**Comment by TheZoq2** [comment]
image
Made some final adjustments. There might be one or two things left that I *could* do, but i'm happy with this for now

**Comment by EMattfolk** [comment]
I was thinking that the arguments should go right under function definitions, so as to get a quick overview of the function. I could do this if you guys think it's a good idea.

**Comment by TheZoq2** [comment]
I think that makes sense

**Comment by TheZoq2** [comment]
Rebased on master now, so this should be ready to merge

**Comment by sornas** [comment]
It looks like not everything got the larger monospace font.

image

**Review by FredTheDino — APPROVED** [review]
LGTM, if you want to fix the final piece of monospace front, that'd be great, unless I do it before you...

**Review by sornas — APPROVED** [review]

**Pull request merged**